native_client/args.h

```diff
@@ -1,7 +1,11 @@
 #ifndef __ARGS_H__
 #define __ARGS_H__
 
+#if defined(_MSC_VER)
+#include "getopt_win.h"
```

Comment:
needs to be named differenly otherwise some tensorflow build will pick-it when building `generate_trie` and others